CHANGELOG.md

```diff
@@ -18,8 +18,9 @@ The types of changes are:
 ## [Unreleased](https://github.com/ethyca/fides/compare/1.7.0...main)
 
 ### Added
-* Add datasets via YAML in the UI [#708](https://github.com/ethyca/fides/issues/708)
-* Add delete confirmation when deleting a field or collection from a dataset [808](https://github.com/ethyca/fides/issues/808)
+* Add datasets via YAML in the UI [#813](https://github.com/ethyca/fides/pull/813)
+* Add datasets via database connection (UI only) [#834](https://github.com/ethyca/fides/pull/834)
+* Add delete confirmation when deleting a field or collection from a dataset [#809](https://github.com/ethyca/fides/pull/809)
```

Comment:
Just fixing some of the links in my other changelog's that were going to issues instead of PR's